docs/development/IntegrationTest.md

```diff
@@ -12,17 +12,13 @@
   limitations under the License. See accompanying LICENSE file.
 -->
 
-# IntegrationTest
+# Integration Test
+## Introduction
+1.  Now, Apache Submarine supports two kinds of integration tests, `test-e2e` and `test-k8s`, and these two modules can be found in the [submarine/submarine-test](https://github.com/apache/submarine/tree/master/submarine-test) directory.
 
-Submarine now supports two kinds of integration tests.
+2.  Currently, there are some differences between `test-e2e` and `test-k8s` in operation mode. To elaborate, `test-e2e` needs to deploy Apache Submarine locally, while `test-k8s` deploys Apache Submarine via k8s.
 
-They are in the project's `submarine/submarine-test` directory, There are two modules, `e2e` and `test-k8s`.
-
-There are currently some differences between `test-e2e` and `test-k8s` in operation mode.
-
-Among them, `test-e2e` needs to deploy submarine locally, while `test-k8s` uses k8s to deploy submarine.
-
-These two different test methods can be applied to different test scenarios. (In the future, these two test methods may be combined or adjusted)
+3.  These two different test methods can be applied to different test scenarios. (In the future, these two test methods may be combined or adjusted)
```

Comment:
How about we change two methods to two test modules

Reply:
Thank you for your review. I have changed "test methods" to "test modules".